toolbox/cijoe-pkg-xnvme/src/cijoe/xnvme/tests/tools/test_xnvme.py

```diff
@@ -118,6 +126,8 @@ def test_log(cijoe, device, be_opts, cli_args):
 
     if be_opts["be"] == "linux" and be_opts["admin"] in ["block"]:
         pytest.skip(reason="[admin=block] does not implement get_log")
```

Comment:
should this be `get-log` instead of get_log?

Reply:
yeah, changed to "get-log".